### ISSUE TITLE
1: waarneembaar (perceivable)

### ISSUE THREAD

**Issue by KyanTG** [KyanTG/all-human-accessible-website#14]
![image](https://github.com/user-attachments/assets/0f57c1f1-5948-408a-bbd5-f72a3ff929d4)

Ik heb de verschillende kleuren getest die er gebruikt zijn, de gebruikte blauw in het design werkt goed op de witte achtergrond. Het enige probleem is dat WCAG AAA failt.
